jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -189,6 +189,7 @@ class PlotOptions(PluginTemplateMixin):
     stretch_vmax_sync = Dict().tag(sync=True)
 
     stretch_hist_zoom_limits = Bool().tag(sync=True)
+    stretch_hist_bin_size = Int().tag(sync=True)

```

Comment:
could `bin_size` cause anyone to think it means "width" instead of number of bins?  Maybe `stretch_hist_nbins` would avoid that?

Reply:
Agreed, I think that makes more sense.